releases/0.3.0.heads.md

```diff
@@ -0,0 +1,134 @@
+# Dasharo Trustworthy Computing
+
+[https://dasharo.com](https://dasharo.com)
+
+Talos II support in Heads.
+
+## v0.3.0 Heads
+
+This release contains changes only for Heads.
+There are no coreboot changes included in this release.
+
+### Release Changelog
+\
+**Added:**
+* Support for PPC64 architecture - first non-x86 target for Heads.
+* Improve project structure introducing subdirectory per architecture.
+* Replace some hard-coded paths with variables.
+
+### Upstream status
+
+#### Pending
+
+Lines added: 14843
+Lines removed: 25
+
+#### Active
+
+* [55038: src/lib/fmap.c: use le*toh() functions where needed](https://review.coreboot.org/c/coreboot/+/55038)
+* [55039: util/cbfstool/flashmap/fmap.c: fix fmaptool endianness bugs on BE](https://review.coreboot.org/c/coreboot/+/55039)
+
+Lines added: 49
+Lines removed: 38
+
+#### Merged
+
+* [55037: ppc64/byteorder.h: define use of big endian](https://review.coreboot.org/c/coreboot/+/55037)
+* [55040: commonlib/lz4_wrapper.c: do not use bitfields](https://review.coreboot.org/c/coreboot/+/55040)
+* [55041: docs/flashmap: state the endianness of FMAP](https://review.coreboot.org/c/coreboot/+/55041/3)
+
+Lines added: 36
+Lines removed: 41
+
+### Download firmware images
+
+#### Release binaries
+
+* [dasharo-trustworthy-computing-v0.3.0.rom]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.SHA256]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.SHA256.sig]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc.SHA256]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc.SHA256.sig]()
+
+See how to verify signatures on
+
+[![asciicast](https://asciinema.org/a/XTwZHG3RMWe0Mm2VrV7VPloYB.svg)](https://asciinema.org/a/XTwZHG3RMWe0Mm2VrV7VPloYB)
+
+### Release statistics
+
+In this release release: \
+<span style="color:yellow">153</span> files were changed \
```

Comment:
@IgorBagnucki Statistics for coreboot repository does not make much sense

Reply:
@These are statistics from commits to Heads. I'll emphasize it